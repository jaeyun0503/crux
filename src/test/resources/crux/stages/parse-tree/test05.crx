
// test declaration

let global: int;
let a: [bool; 5];

fn main() {

}
