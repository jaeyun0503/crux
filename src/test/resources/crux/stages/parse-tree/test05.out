program
  declList
    decl
      varDecl
        type
    decl
      arrayDecl
        type
    decl
      functionDefn
        paramList
        stmtBlock
          stmtList
